### ISSUE TITLE
Carbon 10 Toggle WCGA 2.1 Guideline 1.4.1 violation

### ISSUE THREAD

**Issue by Majavita** [carbon-design-system/carbon#8497]
1.4.1 Use of Color Level A violation by C10 Toggle

## Environment

> Browser

Firefox

## Detailed description

> What version of the Carbon Design System are you using?

10.25 

> What did you expect to happen?

The toggle to still be visible with at least an outline

> What happened instead?

Toggle completely disappeared, leaving only the text.

> What WCAG 2.1 checkpoint does the issue violate?

1.4.1 

## Steps to reproduce the issue

Issue can be recreated using the Carbon Design Systems Live Demo:
https://www.carbondesignsystem.com/components/toggle/usage

In Firefox:
Remove color using the Preferences>Language and Appearance>Fonts and Colors preferences 

![image](https://user-images.githubusercontent.com/32313977/116150380-e7497680-a6a8-11eb-8ab6-a07509812d00.png)

Before color removal:
![image](https://user-images.githubusercontent.com/32313977/115927413-ff6f8a80-a449-11eb-9bc1-c07e1c934038.png)

After color removal:
![image](https://user-images.githubusercontent.com/32313977/115927259-c6cfb100-a449-11eb-8ade-823583af8553.png)

**Comment by joshblack** [comment]
Hi there @Majavita! 👋  Could you provide more information on how to reproduce? I wasn't sure how to use the steps that you provided, are these OS-level instructions or potentially browser-level instructions? Thanks!

**Comment by dakahn** [comment]
This isn't a 1.4.1 violation. 1.4.1 is concerned with making sure users can see information conveyed by color differences. This doesn't fall under that category. Closing, but i can reopen if there's something I'm missing.

**Comment by Majavita** [comment]
The way our AVT interpret "Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element." is a bit different and there is room for improvement. If we remove color from a page, the visual element should still be distinguishable.  If the toggle has an outline we would be able to see if it is set to left or right, off/on, etc.  We do get additional info with the text, which is great, but I would think we would want to still be able to see the visual element.  @hakahn how do you normally check components for color removal?   Hi, @joshblack I added a screenshot to the steps - hope it helps.

**Comment by dakahn** [comment]
> Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element.

Exactly, yeah. During an audit you might remove color from the page to test if various warnings/errors/or other states are still understandable and perceivable. The stroke on an SVG going from green to red as the only means of conveying an incorrect user input (no accompanying warning message) is a 1.4.1 violation as color blind users might _miss_ the color change altogether.

That said to my knowledge there is no WCAG guideline that mandates general use accessibility when color is removed from the browser and so we don't support this use case.

**Comment by Majavita** [comment]
@joshblack @dakahn there seems to be disagreement on how to satisfy this requirement.  Please refer to https://www.ibm.com/able/requirements/requirements/#1_4_1, and for the instruction for removing color, please refer to https://pages.github.ibm.com/IBMa/able/Test/Manual_tests/web/#configuration-instructions-for-visual-tests.

**Comment by Majavita** [comment]
Accessibility team conducted a review and will update their Manual Test guidance on how to check 1.4.1 compliance:  

"The test guidance is outdated. The proper way to test this is to evaluate the page grayscaled to ensure that color isn’t required to distinguish between colored elements - not to override colors."

"Both Windows (https://winaero.com/enable-greyscale-mode-windows-10/) and Mac (https://9to5mac.com/2019/05/27/how-to-use-grayscale-on-your-mac/) now support grayscale modes.".

The toggle does not disappear when user enables grayscale.
